webgpu.h

```diff
@@ -444,6 +444,16 @@ typedef enum WGPUBufferUsage {
 } WGPUBufferUsage;
 typedef WGPUFlags WGPUBufferUsageFlags;
 
+typedef enum WGPUFeatureName {
```

Comment:
nit: should we reserve 0 for undefined even though I'm pretty sure we'll never use it? It at least means default-initialization will error instead of enabling DepthClamping.

Reply:
That would make sense to me. Would the same argument apply for `WGPUTextureDimension_1D`, `WGPUTextureComponentType_Float`, (everything else?)

Reply:
I think so, I think we agreed to do that... somewhere.

Reply:
Ok, we can do that in a separate PR if that's the decision. I think *maybe* it's not because we wanted zero-initialization of other structs ot have reasonable defaults...? maybe.

Reply:
We discussed it in https://github.com/webgpu-native/webgpu-headers/issues/45 but didn't make the change quite yet.